tests/Query/QueryStringTest.php

```diff
@@ -17,7 +17,7 @@ class QueryStringTest extends BaseTest
      */
     public function testSearchMultipleFields(): void
     {
-        $str = \md5(\mt_rand());
+        $str = uniqid();
```

Comment:
This change is because `\md5()` expects a `string` and `\mt_rand()` returns an `int`. The return value of `\mt_rand()` could also be casted to `string`, but instead I've used `uniqid()`, if cast is preferred, let me know.